webapp/package.json

```diff
@@ -5,86 +5,66 @@
+    "fix": "eslint --ignore-pattern node_modules --ignore-pattern dist --ext .js --ext .jsx --ext tsx --ext ts ./src --quiet --fix --cache",
     "test": "jest --forceExit --detectOpenHandles --verbose",
     "test:watch": "jest --watch",
     "test-ci": "jest --forceExit --detectOpenHandles --maxWorkers=2",
     "check-types": "tsc"
   },
   "devDependencies": {
-    "@babel/cli": "7.16.8",
-    "@babel/core": "7.16.12",
-    "@babel/plugin-proposal-class-properties": "7.16.7",
-    "@babel/plugin-proposal-object-rest-spread": "7.16.7",
-    "@babel/plugin-proposal-optional-chaining": "7.16.7",
-    "@babel/plugin-syntax-dynamic-import": "7.8.3",
-    "@babel/preset-env": "7.16.11",
-    "@babel/preset-react": "7.16.7",
-    "@babel/preset-typescript": "7.16.7",
-    "@babel/runtime": "7.16.7",
-    "@emotion/babel-preset-css-prop": "11.2.0",
-    "@emotion/core": "10.3.1",
-    "@mattermost/types": "6.7.0-0",
-    "@testing-library/jest-dom": "5.16.1",
-    "@types/babel__core": "7.1.18",
-    "@types/babel__template": "7.4.1",
-    "@types/enzyme": "3.10.11",
-    "@types/jest": "27.4.0",
-    "@types/node": "17.0.12",
-    "@types/react": "16.14.26",
-    "@types/react-dom": "17.0.11",
-    "@types/react-redux": "7.1.22",
-    "@types/react-router-dom": "5.1.5",
-    "@types/react-transition-group": "4.4.0",
-    "@typescript-eslint/eslint-plugin": "5.10.1",
-    "@typescript-eslint/parser": "5.10.1",
-    "@typescript-eslint/typescript-estree": "5.52.0",
-    "babel-eslint": "10.1.0",
-    "babel-loader": "8.2.2",
-    "babel-plugin-formatjs": "10.3.7",
-    "babel-plugin-typescript-to-proptypes": "2.0.0",
-    "css-loader": "6.5.1",
-    "enzyme": "3.11.0",
-    "enzyme-adapter-react-16": "1.15.6",
-    "enzyme-to-json": "3.6.2",
-    "eslint": "8.8.0",
+    "@testing-library/jest-dom": "6.4.2",
+    "@types/jest": "29.5.12",
+    "@types/node": "20.11.20",
+    "@types/react": "18.2.58",
+    "@types/react-dom": "18.2.19",
+    "@types/react-redux": "7.1.33",
+    "@types/react-router-dom": "5.3.3",
+    "@types/react-transition-group": "4.4.10",
+    "@typescript-eslint/eslint-plugin": "7.0.2",
+    "@typescript-eslint/parser": "7.0.2",
+    "@typescript-eslint/typescript-estree": "7.0.2",
+    "css-loader": "6.10.0",
+    "eslint": "8.57.0",
     "eslint-import-resolver-alias": "1.1.2",
-    "eslint-import-resolver-typescript": "2.7.1",
-    "eslint-import-resolver-webpack": "0.13.2",
-    "eslint-plugin-babel": "5.3.1",
-    "eslint-plugin-cypress": "2.12.1",
+    "eslint-import-resolver-typescript": "3.6.1",
+    "eslint-import-resolver-webpack": "0.13.8",
+    "eslint-plugin-cypress": "2.15.1",
     "eslint-plugin-header": "3.1.1",
-    "eslint-plugin-import": "2.25.4",
-    "eslint-plugin-jest": "26.5.3",
+    "eslint-plugin-import": "2.29.1",
+    "eslint-plugin-jest": "27.9.0",
     "eslint-plugin-mattermost": "github:mattermost/eslint-plugin-mattermost",
```

Comment:
Also, that'd mean you can get rid of the old ESLint plugin which is a git dependency and causes NPM to complain because of a non-existent vulnerability in it